include/libtorrent/torrent_status.hpp

```diff
@@ -460,6 +460,7 @@ namespace libtorrent {
 		// was saved.
 		bool need_save_resume = false;
 
+#ifndef TORRENT_NO_DEPRECATE
```

Comment:
I was waffling about making the change to torrent_status because I really wanted my other change (#1937) to go to RC_1_1, not master. If I could get *this* change to RC_1_1 I could rebase #1937 after it.